src/rwrapr/renv.py

```diff
@@ -212,13 +212,13 @@ def rclass(self, x: Any) -> RReturnType:
         foo: Callable[..., RReturnType] = rfunc("class")
         return foo(x)
 
-    def reval(self, expr: str, rview: bool) -> Any:
+    def reval(self, expr: str, rview: bool | None = None) -> Any:
```

Comment:
Why None? Shouldn't it be `review: bool = False`? The description of the argument says "Defaults to False" and does not match the current line?

Reply:
The description was wrong. It does not default to False, but None. When `rview` is `None` the current setting of `wr.settings.rview_mode` is used. I've submitted a new commit with the updated doc-string.